Project 2 (13.12)/script.js

```diff
@@ -0,0 +1,95 @@
+// exercise 1
+
+// Try 1
+function reverseText(string) {
+	if (typeof string === 'string') {
+		let newString = '';
+		for (let i = string.length - 1; i >= 0; i--) {
+			newString += string[i];
+		}
+		console.log(newString);
```

Comment:
mała uwaga - jeżeli w poleceniu mówimy, że funkcja ma coś zwrócić, to wynikiem ma być return, a nie console.log :)